src/averageAges.js

```diff
@@ -7,19 +7,31 @@
  * function calculates average age only for men who died in this century
  *
  * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
+ * Divide year of human's death by 100: Math.ceil(human.died / 100)
  *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+
+function averageAge(people) {
```

Comment:
function name should describe the action it does, so it should start with verb.
Examples: `get...`, `set...`, `calculate...` and so on